README.mdx

```diff
@@ -26,7 +26,7 @@ itself is published at [`docs.linea.build`](https://docs.linea.build/).
 See something missing? Error in our documentation? Create an issue [here](https://github.com/Consensys/doc.linea/issues).
 
 Alternatively, help us improve our documentation! [Fork our repo](https://github.com/ConsenSys/doc.linea/fork),
-create a pull request, and tag us for review. (For help on this, see [below](#how-to-submit-a-suggestion-or-change)).
+create a pull request, and tag us for review. (For help on this, see [below](#Contribute-to-the-Zero-Knowledge-Glossary)).
```

Comment:
```suggestion
create a pull request, and tag us for review.
```

I think we must have removed the relevant section, and now we don't have any text describing how to create a PR and tag us for review. So we can remove.